.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ workflows:
 jobs:
   test:
```

Comment:
Don't we still need the environment variable for this one?

```
    environment:
      CI_E2E_FILENAME: "494948f4/rel-nightly"
```

Reply:
actually, we still need to upload an artifact to the path  "<commit-hash>/rel-nightly" and in e2elive it will determine the correct commit hash to use based on the submodule.